participant/bin/prod-start.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
do we also need to change the shebang in bin/run-e2e.sh?

Reply:
I don't believe so. I think this issue is specifically around AWS hosted postgres and we aren't running any e2e on AWS